registration-default.properties

```diff
@@ -475,12 +475,18 @@ mosip.kernel.otp.expiry-time=${mosip.kernel.otp.expiry-time}
 
 ## backward compatibility fields	
 mosip.registration.uin.update.configured.fields=name,age,gender,address,phone,email,parentOrGuardianDetails,foreigner,biometrics,cnieNumber
-mosip.primary-language=${mosip.primary-language}	
-mosip.secondary-language=${mosip.secondary-language}	
+mosip.primary-language=eng
+mosip.secondary-language=ara
```

Comment:
why the existing value is ${mosip.primary-language}, is this suppose to be configured in config map of config server ?

Reply:
The place-holder mentioned, is not present in application-default.properties to fill in the values. If you suggest to add them there, will add those two properties in application-default.properties @vishwa-vyom